ta/pkcs11/src/pkcs11_token.h

```diff
@@ -182,7 +182,6 @@ struct pkcs11_session {
 	struct ck_token *token;
 	enum pkcs11_mechanism_id handle;
 	struct object_list object_list;
-	struct handle_db object_handle_db;
```

Comment:
There is a comment at line 178  * @object_handle_db - Database for object handles published by the session which also needs to be removed.

Reply:
Right. Removing.